oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,450 @@
+nominating, or voting for other employees of their company.  This
+would be impractical as, currently, single companies are locus of
+component maintenance work. This may be addressed in future revisions
+to this proposal.
+
+Unmaintained Components
+-----------------------
+In the case of unmaintained components, nominations will be made to
+the scrum-of-scrums to fill the vacancy.  Nominations can be made by
+any member of the community including the nominee themselves.  If
+there is no objection from existing maintainers, the role will be
+conferred.
+
+If consensus cannot be reached lazily [#lazy_concensus]_, a vote of existing maintainers
+will be taken.  Maintainers can approve a nomination with the approval
+of at least three quarters of the members.
+
+New Components
+--------------  
+In the case of new repositories, there should be a designated
+maintainer named before the repository is created.
+
+Removing a Maintainer
+---------------------
+From time-to-time it will be necessary to remove a component
+maintainer.  This might happen because:
+
+* A maintainer leaves the project or changes employment
+* A maintainer does not have the time or interest to continue in the
+  role
+* A maintainer is unreachable
+* A maintainer is not fulfilling the responsibilities of the role
+
+Trigger events for removing a maintainer would be:
+
+* A maintainer informs the project, the scrum-of-scrums, and/or the `program
+  administrators`_ that they can no longer continue in the role
+* A maintainer has been unreachable for more than 30 days
+* A maintainer has violated the project `Code of Conduct`_
+* A maintainer hasn't met project quality standards consistently
+  despite feedback
+
+In the case that a maintainer has violated the `Code of Conduct`_, the
+Goverance Team, specified in the Code of Conduct, will decide what the
+repercussions of the violation will be, including whether the violator
+can continue in the role of maintainer.  It should be noted that
+maintainers are expected to be exemplars and, as such, will be held to
+a high standard.
+
+In the case that a maintainer is not meeting project quality standards
+consistently, the scrum-of-scrums will inform the maintainer with the
+expectation that their components meet standards within a calendar
+month.  If, after a month, the quality issues are not resolved, the
+maintainer will be removed by the scrum-of-scrums.  Additionally, a
+maintainer who is warned three (3) times in a single calendar year
+will also be removed. The `program administrators`_ serve as a
+backstop - any community member may contact them at any time if
+a problem arises and the normal channels of communication are
+not working.
+
+Documenting Maintainers
+-----------------------
+As it is essential that maintainers are discoverable and reachable,
+the person or persons responsible for maintaining a component should
+be listed in the repository README file along with the mechanism for
+contacting a maintainer.
```

Comment:
At edX there developed an idea of "owning ownership". That is, someone needed to define what ownership meant, keep the documentation of owners up-to-date, and maintain repository health monitoring mechanisms, and facilitate ownership transfers. The grand Ownership Spreadsheet didn't happen until a few folks (such as @feanil and the rest of Arch-BOM) took on that mantle of meta-ownership.

I think this OEP needs a similar person or group of people to, uh, "maintain maintainership". Will that be you, @e0d? Or perhaps tCRIL, or a particular working group?

Reply:
I think this is a service that tCRIL provides or ensures is provided.  Ideally the mechanisms are owned by the whole community, but it's clear that it's someones job at the end of the day.